bonus_file_with_defs.rb

```diff
@@ -0,0 +1,274 @@
+#################################################################################
+###########################    General methods    ###############################
+#################################################################################
+require 'fileutils'
+
+def puts_something
+  puts ""
+  puts "---List of words that you will meet in tests---"
+end
+
+def read_this_file (file_to_read)
+  @array_with_strings_from_txt = []
+  File.open(file_to_read) do |review_file|
+    @array_with_strings_from_txt = review_file.readlines
+    puts @array_with_strings_from_txt
+  end
+end
+
+def start_test_1_or_not
+  puts " "
+  puts "Do you want to continue making tests or still need to learn this words better?(yes/no)"
+  variable = gets.chomp
```

Comment:
It will be better if you change the name of this variable. The name of variable has to explain why we use it. The names kind a "int", "array", "bool" etc. are unclear. We can't write the types of variables in there's name.